### ISSUE TITLE
Pass manually over all existing sources and check for extra information

### ISSUE THREAD

**Issue by baryluk** [openZH/covid_19#295]
```
AG 2020-03-28T18:00     364       3 OK 2020-03-28T19:05:10+01:00 # Extras: ncumul_hosp=50,ncumul_ICU=12,ncumul_vent=9 # URLs: https://www.ag.ch/de/themen_1/coronavirus_2/alle_ereignisse/alle_ereignisse_1.jsp
AI 2020-03-27T18:00      12       - OK 2020-03-28T19:05:14+01:00 # URLs: https://www.ai.ch/themen/gesundheit-alter-und-soziales/gesundheitsfoerderung-und-praevention/uebertragbare-krankheiten/coronavirus
AR 2020-03-28T08:00      45       2 OK 2020-03-28T19:05:18+01:00 # URLs: https://www.ar.ch/verwaltung/departement-gesundheit-und-soziales/amt-fuer-gesundheit/informationsseite-coronavirus/
BE 2020-03-28T          767       9 OK 2020-03-28T19:05:23+01:00 # URLs: https://www.besondere-lage.sites.be.ch/besondere-lage_sites/de/index/corona/index.html
BL 2020-03-28T          502       5 OK 2020-03-28T19:05:32+01:00 # URLs: https://www.statistik.bl.ch/files/sites/Grafiken/COVID19/Grafik_COVID19_BL_Linie.htm
BS 2020-03-28T10:00     573       - OK 2020-03-28T19:05:45+01:00 # URLs: https://www.gd.bs.ch/, https://www.gd.bs.ch//nm/2020-tagesbulletin-coronavirus-573-bestaetigte-faelle-im-kanton-basel-stadt-gd.html
FR 2020-03-28T          421      15 OK 2020-03-28T19:05:52+01:00 # Extras: ncumul_hosp=67 # URLs: https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
GE 2020-03-27T12:00    1924      23 OK 2020-03-28T19:06:06+01:00 # Extras: ncumul_hosp=313,ncumul_ICU=54 # URLs: https://www.ge.ch/document/point-coronavirus-maladie-covid-19/telecharger
GL 2020-03-28T13:30      47       - OK 2020-03-28T19:06:08+01:00 # Extras: ncumul_hosp=3 # URLs: https://www.gl.ch/verwaltung/finanzen-und-gesundheit/gesundheit/coronavirus.html/4817
GR 2020-03-27T          409       9 OK 2020-03-28T19:06:16+01:00 # URLs: https://www.gr.ch/DE/institutionen/verwaltung/djsg/ga/coronavirus/info/Seiten/Start.aspx
JU 2020-03-28T16:00     118       - OK 2020-03-28T19:06:32+01:00 # URLs: https://www.jura.ch/fr/Autorites/Coronavirus/Accueil/Coronavirus-Informations-officielles-a-la-population-jurassienne.html, https://www.jura.ch/fr/Autorites/Coronavirus/Chiffres-H-JU/Evolution-des-cas-COVID-19-dans-le-Jura.html
LU 2020-03-28T11:00     317       4 OK 2020-03-28T19:06:41+01:00 # URLs: https://gesundheit.lu.ch/themen/Humanmedizin/Infektionskrankheiten/Coronavirus
NE 2020-03-28T14:00     311       6 OK 2020-03-28T19:06:51+01:00 # URLs: https://www.ne.ch/autorites/DFS/SCSP/medecin-cantonal/maladies-vaccinations/Pages/Coronavirus.aspx
NW 2020-03-28T15:30      55       0 OK 2020-03-28T19:06:59+01:00 # URLs: https://www.nw.ch/gesundheitsamtdienste/6044
OW 2020-03-27T           37       - OK 2020-03-28T19:07:09+01:00 # URLs: https://www.ow.ch/de/verwaltung/dienstleistungen/?dienst_id=5962
SG 2020-03-28T          339       5 OK 2020-03-28T19:07:14+01:00 # URLs: https://www.sg.ch/tools/informationen-coronavirus.html
SH 2020-03-28T07:30      37       - OK 2020-03-28T19:07:20+01:00 # URLs: https://sh.ch/CMS/content.jsp?contentid=3209198&language=DE&_=1584807070095
SO 2020-03-28T00:00     173       1 OK 2020-03-28T19:07:26+01:00 # URLs: https://corona.so.ch/
SZ 2020-03-28T          122       - OK 2020-03-28T19:07:33+01:00 # Extras: ncumul_released=33 # URLs: https://www.sz.ch/behoerden/information-medien/medienmitteilungen/coronavirus.html/72-416-412-1379-6948
TG 2020-03-28T          134       - OK 2020-03-28T19:07:39+01:00 # URLs: https://www.tg.ch/news/fachdossier-coronavirus.html/10552
TI 2020-03-28T08:00    1727      87 OK 2020-03-28T19:08:04+01:00 # Extras: ncumul_hosp=385,ncumul_ICU=69,ncumul_vent=60 # URLs: https://www4.ti.ch/dss/dsp/covid19/home/, https://www4.ti.ch/area-media/comunicati/, https://www4.ti.ch/area-media/comunicati/dettaglio-comunicato/?NEWS_ID=187538&tx_tichareamedia_comunicazioni%5Baction%5D=show&tx_tichareamedia_comunicazioni%5Bcontroller%5D=Comunicazioni&cHash=736a7f1a780ed9bdb1d8e45f8a0326ff
UR 2020-03-27T12:00      48       0 OK 2020-03-28T19:08:08+01:00 # Extras: ncumul_released=3,ncumul_hosp=7 # URLs: https://www.ur.ch/themen/2962
VD 2020-03-27T         2745      48 OK 2020-03-28T19:08:09+01:00 # Extras: ncumul_released=174,ncumul_hosp=370 # URLs: https://api.datawrapper.de/v3/charts/tr5bJ/data
VS 2020-03-28T          902      21 OK 2020-03-28T19:08:17+01:00 # URLs: https://www.vs.ch/de/web/coronavirus
ZG 2020-03-27T18:00     101       1 OK 2020-03-28T19:08:20+01:00 # Extras: ncumul_released=18 # URLs: https://www.zg.ch/behoerden/gesundheitsdirektion/amt-fuer-gesundheit/corona
ZH 2020-03-28T14:00    1720      15 OK 2020-03-28T19:08:25+01:00 # Extras: ncumul_hosp=187,ncumul_vent=42 # URLs: https://gd.zh.ch/internet/gesundheitsdirektion/de/themen/coronavirus.html
```

As can be seen, we now extract and parse some extra information for some cantons, like numbers for hospitalization, ICU, ventilated, recovered/released.

Right now we have this data for:
  * AG, FR, GE, GL, SZ, TI, UR, VD, ZG, ZH
  * And very soon for JU and GR (PRs pending).

The tracker here is to look at the other cantons, and see if there is any new information. We have people periodically looking at them, and also when scrapes break, but it is possible some cantonal pages started providing more extra information and numbers, and we didn't notice.

So, I think it would be good to check the other cantons, and explicitly say "As of 2020-03-XX there are no information abut X, Y, Z for canton CC on page U".

**Comment by metaodi** [comment]
Let's encourage the people checking the sites periodically to report back if there are numbers they have to put in manually.

cc @andreasamsler

**Comment by metaodi** [comment]
- TG: scraper does not update number of deceased, it's available
- VS: Scraper did not check additional data in pdf https://www.vs.ch/documents/6756452/7008787/Sit%20Epid%20-%20%C3%89tat%20Stand%2028.03.2020.pdf

**Comment by baryluk** [comment]
@metaodi Do you have a link for TG with information about deceased numbers? I don't see it on https://www.tg.ch/news/fachdossier-coronavirus.html/10552

**Comment by metaodi** [comment]
Nevermind: the feedback was based on a news entry in the website, since they have their first death in TG. So it's not yet available in a scrapable form.

**Comment by baryluk** [comment]
* SG: As of 2020-03-28 no extra information at https://www.sg.ch/tools/informationen-coronavirus.html
* LU: As of 2020-03-28 no extra information at https://gesundheit.lu.ch/themen/Humanmedizin/Infektionskrankheiten/Coronavirus
* FL: As of 2020-03-28 no extra information in PDFs linked from https://www.regierung.li/coronavirus
* SO: As of 2020-03-28 no extra information at https://corona.so.ch/
* NE: As of 2020-03-28 no extra information at https://www.ne.ch/autorites/DFS/SCSP/medecin-cantonal/maladies-vaccinations/Pages/Coronavirus.aspx

Extra numbers available:
* AR: As of 2020-03-28 there is number of tested people on https://www.ar.ch/verwaltung/departement-gesundheit-und-soziales/amt-fuer-gesundheit/informationsseite-coronavirus/
* BS: As of 2020-03-28 (and few days before) death numbers are available in sub-articles, like https://www.gd.bs.ch//nm/2020-tagesbulletin-coronavirus-573-bestaetigte-faelle-im-kanton-basel-stadt-gd.html , but they are hard to parse by scraper.

**Comment by baryluk** [comment]
* VS: Extracting additional data from pdfs tracked in https://github.com/openZH/covid_19/issues/295

**Comment by baryluk** [comment]
Could somebody please help me with some other non-covered areas? I think it would be good to have 100% coverage and understanding what data is being provided.

**Comment by baryluk** [comment]
TG: https://www.tg.ch/news/fachdossier-coronavirus.html/10552 got a new data for deaths. Tracked in https://github.com/openZH/covid_19/issues/411 , with PR https://github.com/openZH/covid_19/pull/412  to implement it.

SO: Got hospitalized numbers on https://corona.so.ch/ probably today, 2020-04-03, and it is tracked in https://github.com/openZH/covid_19/issues/391 , with PR https://github.com/openZH/covid_19/pull/409

Still few more cantons needs to be rechecked.

**Comment by baryluk** [comment]
GL: https://www.gl.ch/verwaltung/finanzen-und-gesundheit/gesundheit/coronavirus.html/4817 has now information about deaths. Probably since yesterday.

**Comment by baryluk** [comment]
Few extras for few cantons were covered in some recent PRs.

Just in last 5 days:

* FR: Got data on Released.
* VS: Got data on Hospitalized, and ICU.
* NE: Fix scraper, and get Hospitalized, ICU, and Deceased.
* AG: Got data on Recovered.
* BL: Got Hospitalized, and ICU.
* BS: Extract Deceased from the text.
* OW: Got data for Hospitalized, and Deceased.
* TI: Got data for Recovered.
* AG: Got data for ICU.
* GL: Got data for Deceased.
* SH: Got data on Hospitalized.

That is quite nice.

We also now track the regression, and this is the current status for all cantons:

```python3
matrix = {
  # Note: Please keep the order of cantons and entries.
  'AG': ['Deaths', 'Released', 'Hospitalized', 'ICU', 'Vent'],
  'AI': [],
  'AR': ['Deaths'],
  'BE': ['Deaths', 'Hospitalized', 'ICU', 'Vent'],
  'BL': ['Deaths', 'Released', 'Hospitalized', 'ICU'],
  'BS': ['Deaths', 'Released', 'Hospitalized', 'ICU'],
  'FR': ['Deaths', 'Released', 'Hospitalized', 'ICU'],
  'GE': ['Deaths', 'Hospitalized', 'ICU'],
  'GL': ['Deaths', 'Hospitalized'],
  'GR': ['Deaths', 'Hospitalized'],
  'JU': ['Hospitalized', 'ICU'],
  'LU': ['Deaths', 'Hospitalized', 'ICU'],
  'NE': ['Deaths', 'Hospitalized', 'ICU', 'Vent'],
  'NW': ['Deaths', 'Hospitalized', 'ICU'],
  'OW': ['Deaths', 'Hospitalized'],
  'SG': ['Deaths', 'Released', 'Hospitalized', 'ICU'],
  'SH': ['Deaths', 'Hospitalized', 'ICU'],
  'SO': ['Deaths', 'Hospitalized'],
  'SZ': ['Deaths', 'Released'],
  'TG': ['Deaths', 'Hospitalized', 'ICU'],
  'TI': ['Deaths', 'Released', 'Hospitalized', 'ICU', 'Vent'],
  'UR': ['Deaths', 'Released', 'Hospitalized'],
  'VD': ['Deaths', 'Hospitalized', 'ICU'],
  'VS': ['Deaths', 'Released', 'Hospitalized', 'ICU', 'Vent'],
  'ZG': ['Deaths', 'Released', 'Hospitalized', 'ICU'],
  'ZH': ['Deaths', 'Hospitalized', 'Vent'],
  # 'FL': [],  # No scraper.
}
```

That is all pretty good.

Again, there is a need to go again over the list (just the ones that are not full, like TI, VS and AG, which are complete) and recheck the holes.

**Comment by metaodi** [comment]
Closing this for now since this is a regular chore to add new numbers.
